### PR TITLE
Generate tsv instead of txt via discussion in gitter

### PR DESCRIPTION
**Pull request by lepture**
New format with `source`, `revision`, `code`, `name`.

### CODE REVIEW

File: scripts/parse.py [hunk 1]
```diff
@@ -14,24 +14,24 @@
...
+    (201410, '201504/t20150415_712722.html', False),
+    (201308, '201401/t20140116_501070.html', False),
+    (201210, '201301/t20130118_38316.html', False),
+    (201110, '201201/t20120105_38315.html', False),
+    (201010, '201107/t20110726_38314.html', False),
+    (200912, '201006/t20100623_38313.html', False),
+    (200812, '200906/t20090626_38312.html', False),
+    (200712, '200802/t20080215_38311.html', False),
+    (200612, '200704/t20070411_38310.html', False),
+    (200512, '200410/t20041022_38307.html', True),
```
Comment (tonyseek) [inline scripts/parse.py]: Is it possible to make `200512` be string?

File: scripts/parse.py [hunk 2]
```diff
@@ -93,15 +95,20 @@ def main():
...
         req.encoding = 'utf-8'
         el = fromstring(req.text)
 
-        dirname = os.path.join(sys.argv[1], '%s.txt' % suffix)
+        dirname = os.path.join(sys.argv[1], '%s.tsv' % revision)
         print('--> %s' % dirname, file=sys.stderr)
 
+        source = 'stats'
+        if revision in SAC:
+            source = 'sac'
```
Comment (tonyseek) [inline scripts/parse.py]: What is `sac` =. =

Comment (lepture) [inline scripts/parse.py]: http://www.sac.gov.cn/

Comment (tonyseek) [inline scripts/parse.py]: I suggest to name it "gb". "sac" means something in English.

**Comment by lepture** [comment]
ready for merge?

**Pull request merged**